### ISSUE TITLE
Configuration settings for RoboschoolPong-v0

### ISSUE THREAD

**Issue by ashwinipokle** [openai/roboschool#119]
The official release blog for Roboschool states the following for RoboschoolPong-v1 environment -

"With multiplayer training, you can train the same agent playing for both parties (so it plays with itself), you can train two different agents using the same algorithm, or you can even set two different algorithms against each other."

I have gone through sample code provided in the agent-zoo, but I can't find any details of how to configure agents and the environment in each of these individual settings. 

Can you please give me the details of configuration/environment settings for each of these scenarios - i.e. training agent with self-play and for training multiple agents simultaneously with same algorithm? If I want to train Pong with two agents, how do I pass actions to each one of them ?

Basically, how do I do env.step(action) for Player 1 and Player 2 ?

Any help would be appreciated!

**Comment by olegklimov** [comment]
Hi. Please take a look at the example in zoo folder.
